known-shortcuts/sublime.ts

```diff
@@ -1,3 +1,5 @@
+import { AppParams } from "~/types";
+
 const app = {
```

Comment:
The `app` object should be explicitly typed as `AppParams` to ensure type safety and clarity. This will help in maintaining consistency and understanding of the expected structure. However, there is a bug where the `AppParams` type is being used to cast the `app` object without verification that the `app` object actually conforms to the `AppParams` type. This could lead to runtime errors if the `app` object does not match the expected structure of `AppParams`. It is important to ensure that the `app` object is correctly structured according to the `AppParams` type definition. Consider adding a check to ensure that the `app` object matches the `AppParams` type definition.